student-os/scripts/scaffold_repo.py

```
@@ -51,34 +56,90 @@ def ensure(path: Path, dry_run: bool) -> None:
     path.mkdir(parents=True, exist_ok=True)
 
 
-def write_file(path: Path, content: str, dry_run: bool) -> None:
+def write_file(path: Path, content: str, dry_run: bool) -> bool:
```

Ruff (0.16.2)

[warning] Boolean-typed positional argument in function definition

(FBT001)